lib/player.rb

```diff
@@ -50,5 +51,24 @@ def highest_word_score
       highest_scoring_word = ScrabbleMod::Scrabble.highest_score_from(@plays)
       return ScrabbleMod::Scrabble.score(highest_scoring_word)
     end
+
+    def tiles
+      return @player_tiles
+    end
+
+    def draw_tiles(tile_bag)
+      while @player_tiles.length < 7
+        tile = tile_bag.draw_tiles(1)
```

Comment:
The `draw_tiles` method is plural so it would lead me to believe that it would return a list of many tiles